staging/src/k8s.io/apimachinery/pkg/util/net/port_range.go

```diff
@@ -43,35 +43,69 @@ func (pr PortRange) String() string {
 	return fmt.Sprintf("%d-%d", pr.Base, pr.Base+pr.Size-1)
 }
 
+const (
+	SinglePortNotation = iota
```

Comment:
Move this block inside the Set function. These constants doesn't neet to be exported by the package.
```golang
func (pr *PortRange) Set(value string) error {
  const (
```